.github/public/data/AjRadwan.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Anowar J Radwan",
+  "type": "personal",
+  "bio": "Software Developer",
+  "avatar": "https://github.com/AjRadwan.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/AjRadwan",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Anowar_J_Radwan",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/anowarjahanradwan",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on YouTube",
+      "url": "https://www.youtube.com/channel/UCoYxEg_wn4PoJ8e5mAQSotg",
+      "icon": "youtube"
+    },
+    {
+      "name": "Follow me on Facebook",
+      "url": "https://www.facebook.com/AnowarJRadwan",
+      "icon": "facebook"
+    }
+  ]
+}
+
+ 
```

Comment:
Should we fix the endline or let the `github-actions` bot fix it?

Reply:
anything works!

Reply:
I think github-actions should do :)